frontend/src/components/Login.tsx

```diff
@@ -66,17 +66,22 @@ export function OAuthFunction(){
   const OAuth = () => {
     setError(false);
     // Size of popup window
-    const h = 600;
+    const h = 800;
     const w = 500;
     // Center popup window on screen
     const y = window.top.outerHeight / 2 + window.top.screenY - ( h / 2);
     const x = window.top.outerWidth / 2 + window.top.screenX - ( w / 2);
-    const myWindow = window.open("https://example.com", "OAuth Popup",`toolbar=no, location=no, statusbar=no, menubar=no, scrollbars=1, resizable=0, width=${w}, height=${h}, top=${y}, left=${x}`);
+    const myWindow = window.open("http://localhost:8000/hubspot", "OAuth Popup",`toolbar=no, location=no, statusbar=no, menubar=no, scrollbars=1, resizable=0, width=${w}, height=${h}, top=${y}, left=${x}`);
+
     // Show error message when popup window is closed
```

Comment:
For me, after successful oauth with hubspot, I get redirected to the hubspot apps page. Is this the expected behaviour?

Reply:
I added the redirect to go to app.hubspot.com for now just so we could show that the login process is working as expected this will point to the configuration page I think once it's ready!